_tutorials/2016-05-10-iosxr-ansible.md

```diff
@@ -1,36 +1,48 @@
 ---
 permalink: "/tutorials/IOSXR-Ansible"
-author: Bruce McDougall
+author: Mike Korshunov
 excerpt: Getting started with IOSXR and Ansible playbooks
-published: true
+published: false
 title: "IOS-XR Ansible"
 date: "2016-05-16 01:18 +0530"
+tags: 
+  - vagrant
+  - iosxr
+  - cisco
+  - linux
+  - Ansible
 ---
+
+{% include toc icon="table" title="IOS-XR Ansible" %}
+
+
+
 # Introduction
-In this tutorial we are going to cover XRv4 configuration with Ansible. We will setup and try simplest configuration with Unix machine connected to XRv64.
+In this tutorial we are going to cover IOS-XRv configuration with Ansible. We will set up and try a very simple configuration with a Linux machine connected to IOS-XRv.
+
```

Comment:
I think you should add why we're not using the Ansible provisioner directly with Vagrant.Something like:

" The goal of this tutorial is to set up an environment that is identical for Windows, Linux or Mac-OSX users. So instead of setting up Ansible directly on the User's Desktop/Host, we simply spin up an Ubuntu vagrant instance to host our Ansible playbooks and environment. We'll do a separate tutorial on using Ansible directly on Mac-OSX/Windows"